docs/user-guide/api-mediation/configuration-unique-cookie-name-for-multiple-zowe-instances.md

```diff
@@ -8,11 +8,14 @@ To prevent overwriting of the default cookie name in the case of multiple Zowe i
 
 Follow this procedure to configure a unique cookie name for the instances:
 
-1. Open the `zowe.yaml` configuration file.
-2. Find or add the property  `components.gateway.apiml.security.auth.uniqueCookie`, and set it to `true`. A unique cookie name is generated as `apimlAuthenticationToken.cookieIdentifier`.
+1. Open the Zowe YAML configuration file.
+2. Find or add the property  `components.gateway.apiml.security.auth.uniqueCookie`, and set it to `true`.
```

Comment:
If someone needs to add this property, will they know where to add it? If not, should we specify the location?

Reply:
I believe so, but think about this:
"components.gateway.apiml.security.auth" exists here https://github.com/zowe/zowe-install-packaging/blob/v3.x/staging/example-zowe.yaml#L502-L511
So we're saying to add a sub-attribute called "uniqueCookie"

In many places of the doc, we use this "this.that.thing" format for brevity and every time devs interact with users.
I believe it's well understood in general and by enough exposure by now, but the concept is covered in this note in the YAML reference:
https://docs.zowe.org/v2.18.x/appendix/zowe-yaml-configuration

The only thing I can think to do to make it more apparent is to somehow be able to reference that appendix in every place where we make such YAML references. But, I don't think anyone agreed upon a good way to do that if at all.

Reply:
@1000TurquoisePogs -- I think we can avoid linking to the appendix everywhere. We could link to the glossary since the definition includes a link to the appendix. I leave it to you, but my suggestion would be creating a link to the glossary term "Zowe configuration file":

1. Open the [Zowe YAML configuration file](https://docs.zowe.org/stable/appendix/zowe-glossary/#zowe-configuration-file)